druntime/test/profile/bothgc.log.exp

```diff
@@ -1,2 +1,3 @@
 bytes allocated, allocations, type, function, file:line
-          16000	           1000	both.Num both.foo src/both.d:15
+          20496	           1000	Num both.foo src/both.d:15
+          16000	           1000	both.Num core.lifetime._d_newitemT!(Num)._d_newitemT ../../src/core/lifetime.d:2748
```

Comment:
This kind of changes should not happen. They mean the new hook allocates more memory than the previous one, which should not be the case. I'll try to fix them.

Reply:
This was actually incorrect. I later found out this is caused by the call to `GC.qalloc()` within the template hook, which I cannot get past.